### PR TITLE
chore(cleanup): use new playground for every components - no-issue

### PR DESCRIPTION
**Pull request by emeryro**
# PR description

use new playground for every components

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 2d2a6967461eea15a6aa3becb71ce54c421f619e

https://deploy-preview-869--europa-component-library.netlify.com

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 08fa477fd69a430530d73f82d7c25f89831abe66

https://deploy-preview-869--europa-component-library.netlify.com

File: src/systems/ec/specs/components/tag/docs/code.mdx [hunk 1]
```diff
@@ -5,7 +5,11 @@ import Tag from '@ecl/ec-react-component-tag/Tag'
 
 When the click on the tag triggers an action.
 
-<Playground playgroundLink="/storybook/ec/index.html?selectedKind=Tag&selectedStory=as%20a%20button&stories=1">
+<Playground
+  system="ec"
+  selectedKind="Tag"
+  selectedStory="as a button"
```
Comment (yhuard) [inline src/systems/ec/specs/components/tag/docs/code.mdx]: TODO: in the website-comonents, encode URI

File: src/systems/ec/specs/components/text-input/docs/code.mdx [hunk 2]
```diff
@@ -3,7 +3,11 @@ import TextInput from '@ecl/ec-react-component-text-input/TextInput'
 
 import demoContentDefault from '../demo/data--default';
 
-<Playground playgroundLink="/storybook/ec/index.html?selectedKind=TextInput&selectedStory=default&stories=1">
+<Playground
+  system="eu"
```
Comment (yhuard) [inline src/systems/ec/specs/components/text-input/docs/code.mdx]: -> `ec`

File: src/systems/ec/specs/components/text-input/docs/code.mdx [hunk 3]
```diff
@@ -16,7 +20,11 @@ import demoContentDefault from '../demo/data--default';
 
 ## Disabled
 
-<Playground playgroundLink="/storybook/ec/index.html?selectedKind=TextInput&selectedStory=default&stories=1">
+<Playground
+  system="eu"
```
Comment (yhuard) [inline src/systems/ec/specs/components/text-input/docs/code.mdx]: -> `ec`

File: src/systems/ec/specs/components/text-input/docs/code.mdx [hunk 4]
```diff
@@ -30,7 +38,11 @@ import demoContentDefault from '../demo/data--default';
 
 ## With error
 
-<Playground playgroundLink="/storybook/ec/index.html?selectedKind=TextInput&selectedStory=default&stories=1">
+<Playground
+  system="eu"
```
Comment (yhuard) [inline src/systems/ec/specs/components/text-input/docs/code.mdx]: -> `ec`

**Review by yhuard — CHANGES_REQUESTED** [review]

**Review by yhuard — APPROVED** [review]

**Pull request merged**